### DISCUSSION TITLE
Maintenance Release 0.77

### DISCUSSION THREAD

**Discussion by bbidulock** [bbidulock/blackboxwm#39] [General]
This is a maintenance release of the blackboxwm package.  This package contains a maintained fork of the original blackbox CVS that is on Sourceforge.  It includes all of the official blackbox CVS repository, as well as patches collected from pdl-linux, Debian, the blackbox bug list, other GitHub forks.  It also includes greatly enhanced EWMH/ICCCM compliance.  See [COMPLIANCE](COMPLIANCE) in the release for additional information on enhanced compliance.

This release is a maintenance release whose purpose is to update the build system for gcc 10.2 and autoconf 2.71.  There are no outstanding issues at the time of release.

Included in the release is an autoconf tarball for building the release from source.  See the [NEWS](NEWS) and [TODO](TODO) file in the release for more information.  Please report problems to the issues list on [GitHub](https://github.com/bbidulock/blackboxwm/issues).

<hr /><em>This discussion was created from the release <a href='https://github.com/bbidulock/blackboxwm/releases/tag/0.77'>Maintenance Release 0.77</a>.</em>

**Comment by aznlir88** [comment]
Did not expect you guys are still doing it 🤔 as years long fluxbox user i am going to try bb for the first time seriously. Sameone said it is better 🤔
